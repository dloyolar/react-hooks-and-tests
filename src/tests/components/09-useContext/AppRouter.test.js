import React from 'react';

import { mount } from 'enzyme';
import { AppRouter } from '../../../components/09-useContext/AppRouter';
import { UserContext } from '../../../components/09-useContext/UserContext';

describe('Test in <AppRouter />', () => {
  const user = {
    id: 1,
    name: 'Diego',
  };
  const wrapper = mount(
    <UserContext.Provider value={user}>
      <AppRouter />
    </UserContext.Provider>
  );

  test('should be display correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
