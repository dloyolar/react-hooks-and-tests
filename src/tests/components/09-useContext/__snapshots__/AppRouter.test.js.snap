// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <AppRouter /> should be display correctly 1`] = `
<div>
  <nav
    className="navbar navbar-expand-sm navbar-dark bg-dark"
  >
    <a
      className="navbar-brand"
      href="/"
      onClick={[Function]}
    >
      Navbar
    </a>
    <div
      className="collapse navbar-collapse"
      id="navbarNavAltMarkup"
    >
      <div
        className="navbar-nav"
      >
        <a
          activeclassname="active"
          aria-current="page"
          className="nav-item nav-link active"
          exact="true"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
        <a
          activeclassname="active"
          className="nav-item nav-link"
          exact="true"
          href="/about"
          onClick={[Function]}
        >
          About
        </a>
        <a
          activeclassname="active"
          className="nav-item nav-link"
          exact="true"
          href="/login"
          onClick={[Function]}
        >
          Login
        </a>
      </div>
    </div>
  </nav>
  <div
    className="container"
  >
    <div>
      <h1>
        Home Screen
      </h1>
      <hr />
      <pre />
    </div>
  </div>
</div>
`;
